docs/modules/operation/pages/device-config-backup/dcb.adoc

```diff
@@ -17,4 +17,26 @@ NOTE: Backups appear according to their schedule.
 .. In the case of backups that have failed or are in progress, no icon appears.
 . To perform actions such as compare, download, and view history, place a check beside the device(s) and click the menu option.
 
-You can search by node label or IP address and sort the list of backups by typing in the search bar and clicking on the headers in the table respectively.
\ No newline at end of file
+You can search by node label or IP address and sort the list of backups by typing in the search bar and clicking on the headers in the table respectively.
+
+== Monitoring
+
+Events are sent to indicate when backups are started and completed for each monitored device-config service.
+All events are marked with the appropriate node and interface information.
+
+When a backup starts, an event is sent with the following UEI:
```

Comment:
Events information already described in `docs/modules/operation/pages/device-config-backup/configuration.adoc`